README.md

```diff
@@ -147,8 +158,7 @@ the version number to simplify deployment to WebLogic Server.
 
 # Sidecar
 
-The sidecar is a standalone process that runs the exporter. It is primarily intended for use by the
-[WebLogic Server Kubernetes Operator](https://github.com/oracle/weblogic-kubernetes-operator/).
+The sidecar is a standalone process that runs the exporter.
```

Comment:
It is an alternative to using the web-app.

Reply:
Strictly speaking, they are alternatives to each other; I would think that was obvious by the fact that both are listed as options.

Reply:
These are widely separated parts of the documentation - si not so obvious.